t/blackbox.t

```diff
@@ -75,7 +74,6 @@ success!
 --- error_code: 200
 
 
-
```

Comment:
This whitespace was intentional :) openresty projects enforce this to be 3 spaces, so would be good to follow the lead.

Reply:
oh! so, 3 spaces between each test?

Reply:
Yep. Hopefully we will enforce it by some linter at some point.